### ISSUE TITLE
Optimize for Big Picture option not applying (Windows)

### ISSUE THREAD

**Issue by 2ndHouse** [PhilipK/BoilR#199]
I added a non-Steam game through Steam manually and had BoilR pull in the art work for the game. It pulled in the correct art, but it did not apply the Optimize for Big Picture icon correctly. The problem might only apply to games that are already added to Steam? Just started using BoilR and have yet to import launcher games from it.

**Comment by PhilipK** [comment]
Can you send a screenshot of what this looks like?

**Comment by Kingabomination** [comment]
I'm having the same issue on windows. it just keeps the standard icon instead of putting the image I chose. everything else works great.
![steam-Boilr](https://user-images.githubusercontent.com/115125879/194197732-e8168bf2-a74c-4dfd-88f5-180c18f85c96.JPG)

**Comment by PhilipK** [comment]
Can someone try this version?
https://github.com/PhilipK/BoilR/releases/tag/v.1.6.2

It fixes another problem with shortcut icons, but I think it will fix this as well.

**Comment by Kingabomination** [comment]
the import button is missing in version 1.6.2.  ill try the app id fix. 
![Boilr](https://user-images.githubusercontent.com/115125879/194437674-773de6da-e775-4050-8165-7f987fc801ac.JPG)

**Comment by PhilipK** [comment]
You are in the images menu, there is nothing to import there. Is the button
also missing in the import games menu?

**Comment by Kingabomination** [comment]
it is in the import games menu but me and the original poster aren't importing games from other launchers.  we are adding our .exe's manually to steam and using BoilR to add the artwork. there isn't anything to import as far as games go.

**Comment by PhilipK** [comment]
Oh that might be a clue to what is going on.
Also, maybe a "download images" button would also be good.

**Comment by PhilipK** [comment]
Does this only happen for non boilr imported games? Or is it for all games?

**Comment by Kingabomination** [comment]
its for all games. I just imported GOG copies of factorio and secrets of monkey island with the latest release of boilr. 
![image](https://user-images.githubusercontent.com/115125879/196069847-ad312da2-adb3-4682-9b81-ceec496e41c4.png)

**Comment by elkinjosetm** [comment]
This issue is still happening on version v1.7.6. Any way to fix it?

**Comment by PhilipK** [comment]
Have you tried running an import, even if you just have 0 shortcut imports
enabled it should try to fix up shortcuts icon field.

**Comment by elkinjosetm** [comment]
Yes, in fact I'm just trying this on windows for the first time, since I primarily use playnite. I discovered this app to be used for my steam deck, but I figured to also use it on my main gaming PC.

**Comment by Kingabomination** [comment]
honestly Philip, this issue isn't really worth fixing since valve is replacing the old big picture mode with the new steam deck UI. Elkinjosetm you can opt into the new steam beta and get access to the new UI right now. this video will show you how to do it. https://www.youtube.com/watch?v=mMj8CNGKtFc

**Comment by PhilipK** [comment]
Yeah, I must admit that I also knew that was around the corner, so this has not been my nr. 1 priority.
